kube/src/main/java/org/openshift/ping/kube/KubePing.java

```diff
@@ -194,12 +196,16 @@ public void destroy() {
         List<Pod> pods;
         try {
             pods = client.getPods(_namespace, _labels);
+            _hasLoggedPermissionError = false;
+            log.debug("permission flag set to " + _hasLoggedPermissionError);
         } catch (Exception e) {
-            if (log.isWarnEnabled()) {
+            if (log.isWarnEnabled() && (getSystemEnv("LOG_SERVICE_ACCOUNT_PERMISSION_ERROR") != null) && !_hasLoggedPermissionError) {
```

Comment:
Do we need another env to enable/disable this?  I think it's fine to just log the warning.  If folks really want to disable the message, they can modify the properties for the logger, e.g. set it to ERROR instead of WARN or INFO.

Also, we should immediate set the flag to true, to prevent logging again.  If an error happens before then, we'll end up spamming the log again.

As far as the logic goes, it should probably be something like:
```java
if (_hasLoggedPermissionError) {
    _hasLoggedPermissionError = true;
    if (log.isWarnEnabled()) {
        log.warn(...);
    }
}
```